libbpf-tools/fsslower.c

```diff
@@ -451,5 +459,10 @@ int main(int argc, char **argv)
 	perf_buffer__free(pb);
 	fsslower_bpf__destroy(skel);
 
+	if (open_opts.btf_custom_path) {
+		unlink(open_opts.btf_custom_path);
+		free((void *)open_opts.btf_custom_path);
+	}
+
```

Comment:
give this is repeated everywhere, let's have a counter-part to ensure_core_btf() and move this into "cleanup_core_btf()" helper?